lib/bb/client.rb

```diff
@@ -29,7 +33,14 @@ class Client < Common::Client::Base
     # @return [Common::Collection]
     #
     def get_extract_status
-      json = perform(:get, 'bluebutton/extractstatus', nil, token_headers).body
+      if Flipper.enabled?(:mhv_medical_records_migrate_to_api_gateway)
+        BB::Configuration.custom_base_path = APIGW_BASE_PATH
+        json = perform(:get, 'v1/bluebutton/ess/extractstatus', nil, token_headers).body
+      else
+        BB::Configuration.custom_base_path = LEGACY_BASE_PATH
+        json = perform(:get, 'v1/bluebutton/extractstatus', nil, token_headers).body
```

Comment:
I would think that the `v1` isn't necessary here because it's not used currently. Is it intentionally added here and throughout the rest of the file in the disabled flipper scenarios?
```suggestion
        json = perform(:get, 'bluebutton/extractstatus', nil, token_headers).body
```

Reply:
@rmtolmach both the current MHV endpoints _and_ the new API Gateway endpoints contain the `v1` pattern, although there are other differences in the path. I have tested all the endpoints (against staging) with the flag on and off to ensure that we are successfully hitting the correct resources in both scenarios.

Reply:
If you've tested it and it works, that sounds good.